changelogs/fragments/343-telekom-mms-role.yml

```diff
@@ -0,0 +1,2 @@
+major_changes:
```

Comment:
I'm not sure this is a major-change, but rather a minor.

https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment

What do you think?

Reply:
You're right - now that I think about it, it's not a major change... Adding the role doesn't affect existing collection code or user code.